viz_scripts/generic_metrics.ipynb

```diff
@@ -56,21 +57,27 @@
     "%matplotlib inline"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eaf90438",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Do not run this notebook if it has survey_info; nbclient will run up through this cell\n",
```

Comment:
We can't filter on if survey info exists - 

Some deployments have trip-level surveys: [`dfc-fermata`](https://github.com/e-mission/nrel-openpath-deploy-configs/blob/9faf8be480c12b44f2d8673f128567353255b4fe/configs/dfc-fermata.nrel-op.json#L39)

And others have records of the onboarding survey: ['cortezebikes`](https://github.com/e-mission/nrel-openpath-deploy-configs/blob/9faf8be480c12b44f2d8673f128567353255b4fe/configs/cortezebikes.nrel-op.json#L44)

Reply:
Addressed in https://github.com/e-mission/em-public-dashboard/pull/152/commits/468d40ed6279c4acca1d24641366869a9e29594e by filtering as 
```
if not survey_info.get('trip-labels', None) == 'MULTILABEL':
  \\ Do not execute the generic_metrics notebook.
```